### PR TITLE
Add max results to sfctl nodes list

### PR DESCRIPTION
**Pull request by Christina-Kang**
Verifed:

- [ ] Build and test CI passes
- [ ] History and readme updated to reflect changes
- [ ] Package version updated according to semantic versioning rules
- [ ] Tests modified or added, when applicable
- [ ] Updated code owners file, when applicable
- [ ] Read the [PR checklist](https://github.com/Azure/service-fabric-cli/wiki/Checklist)

### CODE REVIEW

File: src/sfctl/tests/request_generation_test.py [hunk 1]
```diff
@@ -331,10 +331,10 @@ def paths_generation_helper(self):  # pylint: disable=too-many-statements
             '/Nodes/nodeName',
             ['api-version=6.0'])
         self.validate_command(  # list
-            'sfctl node list --continuation-token=nodeId --node-status-filter=up',
+            'sfctl node list --continuation-token=nodeId --node-status-filter=up --max-results=3',
             'GET',
             '/Nodes',
-            ['api-version=6.0', 'ContinuationToken=nodeId', 'NodeStatusFilter=up'])
+            ['api-version=6.3', 'ContinuationToken=nodeId', 'NodeStatusFilter=up', 'MaxResults=10'])
```
Comment (likevi-MSFT) [inline src/sfctl/tests/request_generation_test.py]: Why is it MaxResults=10 and --max-results=3 above? Do they have to match? I can't remember how the validate_command works.

Comment (Christina-Kang) [reply src/sfctl/tests/request_generation_test.py]: Typo, thanks :)

File: src/sfctl/params.py [hunk 2]
```diff
@@ -59,6 +59,9 @@ def custom_arguments(self, _):  # pylint: disable=too-many-statements
     with ArgumentsContext(self, 'application deployed-list') as arg_context:
         arg_context.argument('max_results', type=int)
 
+    with ArgumentsContext(self, 'node list') as arg_context:
```
[the remark this answers is not in the document]
Comment (Christina-Kang) [reply src/sfctl/params.py]: What do you mean it's required now? Thanks!

**Comment by Christina-Kang** [comment]
@samedder I've merged this PR with this one : https://github.com/Microsoft/service-fabric-cli/pull/113 so that CI will pass properly. I'll apply the feedback once you respond to #113 instead. Thanks!

**Comment by samedder** [comment]
Thanks, close this PR then whenever.

**Pull request closed without merging**